django_redis/client/default.py

```diff
@@ -842,7 +838,7 @@ def sdiff(
         *keys: KeyT,
         version: Optional[int] = None,
         client: Optional[Redis] = None,
-    ) -> Set[Any]:
+    ) -> builtins.set[Any]:
```

Comment:
why not just `set`?

Reply:
`set` shadows `django_redis.client.default.DefaultClient.set`. We have the same case with all classes that implements a `.set()`method. I'm open for alternative suggestions.